app/src/main/java/org/mifos/mobile/ui/savings_account_withdraw/SavingsAccountWithdrawScreen.kt

```diff
@@ -0,0 +1,241 @@
+            description = savingsWithAssociations?.clientName ?: ""
+        )
+        Spacer(modifier = Modifier.height(8.dp))
+        MifosTitleDescSingleLineEqual(
+            title = stringResource(id = R.string.account_number),
+            description = savingsWithAssociations?.accountNo ?: ""
+        )
+        Spacer(modifier = Modifier.height(8.dp))
+        MifosTitleDescSingleLineEqual(
+            title = stringResource(id = R.string.withdrawal_date),
+            description = getTodayFormatted()
+        )
+        Spacer(modifier = Modifier.height(16.dp))
+        MifosOutlinedTextField(
+            value = remark,
+            label = R.string.remark,
+            error = remarkFieldError,
+            modifier = Modifier.fillMaxWidth(),
+            supportingText = stringResource(
+                R.string.error_validation_blank,
+                stringResource(R.string.remark)
+            ),
+            onValueChange = {
+                remark = it
+                remarkFieldError = false
+            }
+        )
+        Spacer(modifier = Modifier.height(16.dp))
+        Button(
+            onClick = {
+                if (remark.text.isEmpty()) {
+                    remarkFieldError = true
+                } else {
+                    withdraw.invoke(remark.text)
+                }
+            },
+            modifier = Modifier.fillMaxWidth()
+        ) {
+            Text(text = stringResource(id = R.string.withdraw_savings_account))
+        }
+    }
+}
+
+@Composable
+fun MifosTitleDescSingleLineEqual(
+    modifier: Modifier = Modifier,
+    title: String,
+    description: String
+) {
+    Row(
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.SpaceBetween,
+        modifier = modifier.fillMaxWidth()
+    ) {
+        Text(
+            style = MaterialTheme.typography.labelMedium,
+            color = MaterialTheme.colorScheme.onSurface,
+            text = title,
+            modifier = Modifier
+                .alpha(0.7f)
+                .weight(1f)
+        )
+
+        Text(
+            style = MaterialTheme.typography.bodyMedium,
+            color = MaterialTheme.colorScheme.onSurface,
+            text = description,
+            modifier = Modifier.weight(1f)
+        )
+    }
+}
+
+
+class UiStatesParameterProvider : PreviewParameterProvider<SavingsAccountWithdrawUiState> {
+    override val values: Sequence<SavingsAccountWithdrawUiState>
+        get() = sequenceOf(
+            SavingsAccountWithdrawUiState.Initial,
+            SavingsAccountWithdrawUiState.Error(message = Constants.SOMETHING_WENT_WRONG),
+            SavingsAccountWithdrawUiState.Loading,
+            SavingsAccountWithdrawUiState.Success
+        )
+}
+
+@Preview(showSystemUi = true)
+@Composable
+fun SavingsAccountWithdrawScreenPreview(
+    @PreviewParameter(UiStatesParameterProvider::class) savingsAccountWithdrawUiState: SavingsAccountWithdrawUiState
```

Comment:
You don't need it, remove it this parameter

Reply:
I am using this `@PreviewParameter` to preview all the uiStates in a concise way. Do you want me to explicitly write all different funciton to preview all uiStates?